recipes/pyprctl/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyprctl" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyprctl-{{ version }}.tar.gz
+  sha256: 1fb54d3ab030ec02e4afc38fb9662d6634c12834e91ae7959de56a9c09f69c26
+
+build:
+  skip: True  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools >=40.9.0
+    - wheel
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - pyprctl
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/cptpcrd/pyprctl
+  summary: An interface to Linux's prctl() syscall written in pure Python using ctypes.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jansseni
```

Comment:
Is this a typo?

Reply:
@jakirkham Yes, I am sorry. Can you merge https://github.com/conda-forge/pyprctl-feedstock/pull/6 to fix it ?